Project.toml

```diff
@@ -6,9 +6,12 @@ version = "0.1.9"
 [deps]
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 DocumenterTools = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
+Hadamard = "4a05ff16-5f95-55f4-bb53-bb3f467c689a"
 Krylov = "ba0b0d4f-ebba-5204-a429-3ac8c609bfb7"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
```

Comment:
If you add dependencies, there needs to be an explanation in the PR as to why they are there. Also, this branch needs to be updated based on the current master.

Reply:
I have added a line about the `Hadamard` and `SparseArrays` decency and remove the `StatsBase` dependency.